input/fsh/who-cr-CompositionCaseReportBasic.fsh

```diff
@@ -0,0 +1,261 @@
+* code = $LNC#30525-0
+* value[x] only Quantity
+* valueQuantity.value 1..1
+* valueQuantity from WhoCrValueSetAgeUnits
+
+Profile: WhoCrObservationBirthSex
+Parent: Observation
+Id: who-cr-observation-birth-sex
+Title: "WHO CR Observation Birth Sex"
+Description: "WHO Observation of birth sex for case report"
+* code = $LNC#76689-9
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetSexAtBirth
+
+Profile: WhoCrObservationLabTestDate
+Parent: Observation
+Id: who-cr-observation-lab-test-date
+Title: "WHO CR Observation Lab Test Date"
+Description: "WHO Observation for lab test date"
+* code = http://test.org/obsCode#lab-test-date // TODO: code not defined in questionnaire either
+* value[x] only dateTime
+* valueDateTime 1..1
+
+Profile: WhoCrObservationSymptoms
+Parent: Observation
+Id: who-cr-observation-symptoms
+Title: "WHO CR Observation Symptoms"
+Description: "WHO Observation if the patient has any symptoms for case report"
+* code = http://test.org/obsCode#symptoms // TODO: code not defined in questionnaire either
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetYesNo
+
+Profile: WhoCrObservationOnsetDate
+Parent: Observation
+Id: who-cr-observation-onset-date
+Title: "WHO CR Observation Symptoms Onset Date"
+Description: "WHO Observation for date of symptoms onset"
+* code = $LNC#65222-2
+* value[x] only dateTime
+* valueDateTime 1..1
+
+Profile: WhoCrObservationComorbidities
+Parent: Observation
+Id: who-cr-observation-comorbidities
+Title: "WHO CR Observation Co-morbidities"
+Description: "WHO Observation if the patient has any co-morbidities for case report"
+* code = $LNC#75618-9
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetYesNo
+
+Profile: WhoCrObservationHasTravelled
+Parent: Observation
+Id: who-cr-observation-has-travelled
+Title: "WHO CR Observation Has Travelled"
+Description: "WHO Observation if the patient has travelled in the past 14 days prior to symptom onset for case report"
+* code = $LNC#96542-6
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetYesNo
+
+/**
+ * Examples
+ */
+
+Instance: WhoCrBundleExample
+InstanceOf: Bundle
+Usage: #example
+Title: "Who Cr Bundle Example"
+Description: "Example of a clinical bundle representing a case report"
+* type = #document
+* entry[+].fullUrl = "http://test.org/fhir/Composition/WhoCrCompositionBasicExample"
+* entry[=].resource = WhoCrCompositionBasicExample
+* entry[+].fullUrl = "http://test.org/fhir/Encounter/WhoCrEncounterExample"
+* entry[=].resource = WhoCrEncounterExample
+* entry[+].fullUrl = "http://test.org/fhir/Location/WhoCrLocationExample"
+* entry[=].resource = WhoCrLocationExample
+* entry[+].fullUrl = "http://test.org/fhir/Practitioner/WhoCrPractitionerExample"
+* entry[=].resource = WhoCrPractitionerExample
```

Comment:
@de-laz remove these lines - the resources shouldn't be here as they would already exist in the registry - we are only referencing them - not creating new ones